packages/app-store/zapier/pages/setup/index.tsx

```diff
@@ -105,7 +105,7 @@ export default function ZapierSetup(props: IZapierSetupProps) {
                   <li>You&apos;re set!</li>
                 </Trans>
               </ol>
-              <Link href="/apps/installed" passHref={true}>
+              <Link href="/apps/installed/automation?hl=zapier" passHref={true}>
```

Comment:
This is to simulate the same behavior as `getInstalledAppPath` function which is bypassed by Zapier due to its inner working.